### PR TITLE
Issue 29: Add documentation for the git workflow

### PR DESCRIPTION
**Pull request by oscarpolanco**
# Ticket or issue:

[Issue 29](https://github.com/oscarpolanco/node_react_fullstack/issues/29)

## Description

Add documentation for the git workflow that we are going to follow

### CODE REVIEW

File: sections_text/README.md [hunk 1]
```diff
@@ -4487,3 +4487,28 @@ Here we use some of the classes of the [card](https://materializecss.com/cards.h
 - Finally, test on your browser
 
 Finally, we finished our goals for this app but this is just the beginning of our journey because will be your job to expand and refactor the app. Good look and thanks for reading this.
+
+## Git workflow
+
+On this repository I going to implement a `git workflow` that will allow us to accomplish our goals on a consitent and productive manner using git in the most effective way that we can. I defined the following specifications
```
Comment (iocedano) [inline sections_text/README.md:4493]: Conventional commits is a great tool also, https://www.conventionalcommits.org/en/v1.0.0/
I believe could be useful

File: sections_text/README.md [hunk 2]
```diff
@@ -4487,3 +4487,28 @@ Here we use some of the classes of the [card](https://materializecss.com/cards.h
...
+
+- Our main branch will be call `master`
```
Comment (iocedano) [inline sections_text/README.md:4495]: Adding a graphical representation of the flow could help others quickly and clarify any doubts

File: sections_text/README.md (hunk 1 above)
Comment (oscarpolanco) [reply sections_text/README.md:4493]: Yes; the plan is to add that after adding the linter here we just specify the documentation of the git workflow that we are going to follow

File: sections_text/README.md (hunk 2 above)
Comment (oscarpolanco) [reply sections_text/README.md:4495]: Let me check how I can generate an image of the workflow. Nice catch!!

Comment (oscarpolanco) [reply sections_text/README.md:4495]: Done!!!

**Pull request merged**